src/Compilers/Test/Utilities/CSharp/SemanticModelTestBase.cs

```diff
@@ -220,5 +219,41 @@ protected CompilationUtils.SemanticInfoSummary GetSemanticInfoForTest(string tes
         {
             return GetSemanticInfoForTest<ExpressionSyntax>(testSrc);
         }
+
+        protected string GetOperationTreeForTest<TSyntaxNode>(CSharpCompilation compilation)
+            where TSyntaxNode: SyntaxNode
+        {
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+            SyntaxNode syntaxNode = GetSyntaxNodeOfTypeForBinding<TSyntaxNode>(GetSyntaxNodeList(tree));
+            if (syntaxNode == null)
+            {
+                return null;
+            }
+
+            var operation = model.GetOperationInternal(syntaxNode);
```

Comment:
Yes, and we need to test that `SemanticModel.GetOperation` does give us the expected bound node, lowest or highest. I see that VB and C# seem to have different defaults. C# picks the lowest bound node when there are multiple bound nodes for a syntax (see [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Compilation/CSharpSemanticModel.cs,471) and VB picks the highest (see [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Compilation/SemanticModel.vb,143). I will open a separate work item to track why this difference.